staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -65,7 +65,7 @@ func NewGCEPersistentDiskCSITranslator() InTreePlugin {
 	return &gcePersistentDiskCSITranslator{}
 }
 
-func translateAllowedTopologies(terms []v1.TopologySelectorTerm) ([]v1.TopologySelectorTerm, error) {
+func translateAllowedTopologies(terms []v1.TopologySelectorTerm, key string) ([]v1.TopologySelectorTerm, error) {
```

Comment:
Is there a common file we can pull out this function to since it's no longer specific to gce pd?

Reply:
created a common file to host common keys and logics.